src/components/member-card/index.js

```diff
@@ -45,17 +61,37 @@ const Card = ({ developerInfo, isOptionKey }) => {
       {dev && (
         <SuperUserOptions username={username} showSettings={showSettings} />
       )}
-      <motion.img
-        layoutId={username}
-        src={
-          isMember ? `${img_url}?${Math.random() * 100}` : '/images/Avatar.png'
-        }
-        onError={brokenImageHandler}
-        className={
-          isMember ? classNames.imgContainer : classNames.imgContainerNewMember
-        }
-        alt={username}
-      />
+
+      {isMember && (
+        <motion.img
+          layoutId={username}
+          src={
+            isMember
+              ? `${img_url}?${Math.random() * 100}`
+              : '/images/Avatar.png'
+          }
+          onError={brokenImageHandler}
+          className={
+            isMember
+              ? classNames.imgContainer
+              : classNames.imgContainerNewMember
+          }
+          alt={username}
+        />
+      )}
+
+      {!isMember && (
+        <div
+          className={classNames.new_member_img_absent_container}
+          ref={cardRef}
+        >
+          <p
```

Comment:
instead of setting name in useEffect, can't we make us of helper function which extracts initials here?

Reply:
thanks for the suggestion, made the change